tests/test_pdpbox.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import pandas as pd
+from pdpbox import pdp
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeClassifier
+
+class TestPdpbox(unittest.TestCase):
+    def test_simple_pdp(self):
+        # set up data
+        data = pd.read_csv("/input/tests/data/fifa_2018_stats.csv")
+        y = (data['Man of the Match'] == "Yes")
+        feature_names = [i for i in data.columns if data[i].dtype in [np.int64]]
+        X = data[feature_names]
```

Comment:
you need to add an import statement at the top:
```
import numpy as np
```

Reply:
On a linux machine (cloudtop is good), you can run the following to run the tests against the image the CI server built for this PR:
```
cd /path/to/git/docker-python
./test -i gcr.io/kaggle-images/python:update-pdpbox-pretest -p test_pdpbox.py
```

Reply:
Thanks. The tests pass now